definitions/tools/filter_vcf_somatic_llr.cwl

```diff
@@ -5,7 +5,7 @@ class: CommandLineTool
 label: "use the binomial/llr somatic filter to weed out low confidence variants"
 requirements:
     - class: DockerRequirement
-      dockerPull: mgibio/somatic-llr-filter:v0.3
+      dockerPull: mgibio/somatic-llr-filter:v0.4
```

Comment:
Do we want the very newest version?
```suggestion
      dockerPull: mgibio/somatic-llr-filter:v0.4.1
```

Reply:
weird - I thought I pushed that second commit through to update it...   Yes, thanks for catching that!